src/main/java/supersymmetry/api/block/BlockSinteringBrick.java

```diff
@@ -0,0 +1,53 @@
+package supersymmetry.api.block;
+
+import gregtech.api.block.VariantActiveBlock;
+import net.minecraft.block.SoundType;
+import net.minecraft.block.material.Material;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.entity.EntityLiving;
+import net.minecraft.util.IStringSerializable;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.IBlockAccess;
+import org.jetbrains.annotations.NotNull;
+
+public class BlockSinteringBrick extends VariantActiveBlock<BlockSinteringBrick.SinteringBrickType> {
+
+    public BlockSinteringBrick() {
+        super(Material.ROCK);
+        setTranslationKey("sintering_block");
```

Comment:
Did you mean to use this?
```suggestion
        setTranslationKey("sintering_brick");
```